gax-java/gax-grpc/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+    <difference>
+        <differenceType>7002</differenceType>
+        <className>com/google/api/gax/grpc/InstantiatingGrpcChannelProvider$Builder</className>
+        <method>org.threeten.bp.Duration *</method>
```

Comment:
What's the error, if we don't add this?

Reply:
@suztomo I added the clirr check outputs in the PR description